backend/api/comments.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Tuple
+
+from bson import ObjectId
+from flask import Blueprint, jsonify, request
+from flask_jwt_extended import get_jwt_identity, jwt_required
+
+from backend.database_handler_entity import mongo
+
+comment_api = Blueprint('comment_api', __name__)
```

Comment:
I think all three routes below are not RESTful because it is a mix of semantics of `document` entity and `comment` entity.
I guess the better approach in this case is to move these routes to `document` entity like
`POST /documents/<document_id>/comments` -- create comment for exact document
`[GET, PUT, DELETE] /documents/<document_id>/comments/<comment_id>` -- read, update or delete comment for exact document